main.tf

```diff
@@ -24,9 +24,10 @@ locals {
   ])
   using_local_files = length(local.local_collections) + length(local.local_environments) > 0
   lambda_env_variables = {
-    S3_BUCKET           = local.using_local_files ? aws_s3_bucket.postman_bucket[0].bucket : null
-    POSTMAN_COLLECTIONS = jsonencode(var.postman_collections)
-    POSTMAN_API_KEY     = var.postman_api_key
+    S3_BUCKET              = local.using_local_files ? aws_s3_bucket.postman_bucket[0].bucket : null
+    POSTMAN_COLLECTIONS    = jsonencode(var.postman_collections)
+    POSTMAN_API_KEY        = var.postman_api_key
+    TEST_ENV_VAR_OVERRIDES = base64encode(jsonencode(var.test_env_var_overrides))
```

Comment:
(nit) curious why base64 encoding the override variables?

Reply:
I was over thinking it.